### ISSUE TITLE
Tone doesn't play .mp3 on Cordova (Android)

### ISSUE THREAD

**Issue by norman74** [Tonejs/Tone.js#248]
When I try to load .mp3 file there's no errors or something. The sound is running (checked by google chrome "remote devices") but not playing... I mean debug shows that it's playing but it isn't. Ogg's playing nice. You can say: use .ogg and everything will be ok, but I can't :) I need .mp3

**Comment by tambien** [comment]
I haven't tried combining Cordova and Web Audio for many years. But hopefully there's other people out there that develop Tone.js with Cordova that may be able to help

**Comment by thipages** [comment]
I used Cordova/ToneJS, but cant realize any tests at the moment. Sorry for that!

**Comment by norman74** [comment]
thipages, np. But thanks :)

**Comment by tambien** [comment]
@norman74 any resolution to this issue?

**Comment by norman74** [comment]
@tambien

No, I haven't try to fix that. But I wanted to find an alternative for this, like Howler.js. And for some reason, I have similar problems with it. I think I have to postpone my project 😢

**Comment by markmarijnissen** [comment]
I'm using Tone with Cordova and Crosswalk and have no problems.
